nixos/modules/services/backup/restic.nix

```diff
@@ -376,5 +376,21 @@ in
           timerConfig = backup.timerConfig;
         })
         config.services.restic.backups;
+
+    environment.systemPackages = lib.mapAttrsToList (name: backup: let
+      extraOptions = lib.concatMapStrings (arg: " -o ${arg}") backup.extraOptions;
+      resticCmd = "${backup.package}/bin/restic${extraOptions}";
+    in pkgs.writeShellScriptBin "restic-${name}" ''
+      set -a  # automatically export variables
+      ${lib.optionalString (backup.environmentFile != null) "source ${backup.environmentFile}"}
+      ${lib.optionalString (backup.s3CredentialsFile != null) "source ${backup.s3CredentialsFile}"}
+      RESTIC_CACHE_DIR="/var/cache/restic-backups-${name}"
```

Comment:
For the systemd unit, the restic cache is configured as
```nix
RESTIC_CACHE_DIR = "%C/restic-backups-${name}";
```
which, accordingly to the [specifiers documentation](https://www.freedesktop.org/software/systemd/man/systemd.unit.html#Specifiers), should resolve to the very same path as we're dealing with system units.

What bothers me is that we then have both some duplicate value - which must be updated at two places - as well as a variable which might change somehow in the future, even when such a change is very unlikely due to its impact. Unfortunately I haven't found an easy way to format a systemd path specifier, which could be called.

Thus, maybe we can replace the `%C` specifier in the systemd unit by `/var/cache/` and then drop this special case?

Reply:
> Thus, maybe we can replace the %C specifier in the systemd unit by /var/cache/ and then drop this special case?

Sounds reasonable, but I think currently is also fine since it's very unlikely to change.